resources/install-crictl

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+set -o errexit
+
+version=$(head -c 5 <<< "${kube_version}").0
+
+wget https://github.com/kubernetes-sigs/cri-tools/releases/download/$${version}/crictl-$${version}-linux-amd64.tar.gz
+sudo tar zxvf crictl-$${version}-linux-amd64.tar.gz -C /opt/bin
```

Comment:
Why `sudo`, doesn't this run as root?

Reply:
Also, could we do a checksum on the artifact?

1. Its not been mitm meddled with
2. Also check the download was successful